src/agent/coverage/src/code/tests.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use super::*;
+
+#[test]
+fn test_module_filter_def_include_bool() {
+    let text = r#"{ "module": "abc.exe", "include": true }"#;
+    let def: ModuleRuleDef = serde_json::from_str(text).unwrap();
+
+    assert_eq!(def.module, "abc.exe");
+    assert!(matches!(def.rule, RuleDef::Include { include: true }));
+
+    let text = r#"{ "module": "abc.exe", "include": false }"#;
+    let def: ModuleRuleDef = serde_json::from_str(text).unwrap();
+
+    assert_eq!(def.module, "abc.exe");
+    assert!(matches!(def.rule, RuleDef::Include { include: false }));
+}
+
+#[test]
+fn test_module_filter_def_exclude_bool() {
+    let text = r#"{ "module": "abc.exe", "exclude": true }"#;
+    let def: ModuleRuleDef = serde_json::from_str(text).unwrap();
+
+    assert_eq!(def.module, "abc.exe");
+    assert!(matches!(def.rule, RuleDef::Exclude { exclude: true }));
+
+    let text = r#"{ "module": "abc.exe", "exclude": false }"#;
+    let def: ModuleRuleDef = serde_json::from_str(text).unwrap();
+
+    assert_eq!(def.module, "abc.exe");
+    assert!(matches!(def.rule, RuleDef::Exclude { exclude: false }));
```

Comment:
How does `include: true` and `exclude: false` differ?  Or `exclude: true` and `include: false`?

Reply:
They have identical semantics.